parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -2,22 +2,23 @@
 # Timeout waiting for a file
 ERR_FILE_WATCH_TIMEOUT=6 
 set -x
+
 if [ -f /opt/azure/containers/provision.complete ]; then
-      echo "Already ran to success exiting..."
-      exit 0
+    echo "Already ran to success exiting..."
+    exit 0
 fi
 
-for i in $(seq 1 3600); do
+for i in $(seq 1 120); do
```

Comment:
Need to check with the linux team to verify this decrease is OK.

Reply:
this is probably fine, and won't be needed once we're fully on scriptless (since we won't be relying on cloud-init to write out cse_helpers.sh)

Reply:
This code waits for 60 minutes for `#HELPERSEOF` line to be present in the CSE. And deletes this line after.

In my code I had to run this twice. And it added maximum delay to feedback loop (until the test times out). Cost me a lot time to figure it out.